src/core/src/main/java/org/apache/jmeter/gui/util/JSyntaxTextArea.java

```diff
@@ -112,9 +118,25 @@ public void setText(String t) { }
      * @param jSyntaxTextArea
      */
     private static void applyTheme(JSyntaxTextArea jSyntaxTextArea) {
-        String laf = LookAndFeelCommand.getPreferredLafCommand();
-        if (LookAndFeelCommand.isDark(laf)) {
-            DARCULA_THEME.apply(jSyntaxTextArea);
+        Theme theme;
+        if (LookAndFeelCommand.isDark()) {
+            theme = DARCULA_THEME;
+        } else {
+            theme = DEFAULT_THEME;
+        }
```

Comment:
We probably need to have a notion of "editor theme" vs "UI theme". So far the mapping is very basic.